src/coordinax/_src/frames/register_ops.py

```diff
@@ -1,16 +1,18 @@
 """Register for `coordinax.ops`."""
 
-__all__: list[str] = []
+__all__: tuple[str, ...] = ()
 
 
+from plum import dispatch
+
 from dataclassish import replace
 
 from .coordinate import Coordinate
 from coordinax._src.operators import AbstractOperator
 
 
-@AbstractOperator.__call__.dispatch  # type: ignore[misc]
-def call(self: AbstractOperator, x: Coordinate, /) -> Coordinate:
+@dispatch
+def operate(self: AbstractOperator, obj: Coordinate, /) -> Coordinate:
```

Comment:
Nice! I think we discussed having the "root" `operate` implementation be:
```python
def operate(cls, params, tau, obj):
```
where `tau` is the "proper time" and it's implemented in class method form. But maybe it's possible to implement as a method / replacement of the current `__call__` mechanism? Either way, I think the default should take `(tau, obj)` and there should be a dispatch when only `obj` is passed like:
```python
def operate(cls, params, obj):
    return operate(cls, params, None, obj)
```
Thoughts?